src/main/java/com/dev/cinema/model/CinemaHall.java

```diff
@@ -0,0 +1,70 @@
+package com.dev.cinema.model;
+
+import java.util.Objects;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "cinema_hall")
+public class CinemaHall {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private Integer capacity;
+    private String description;
+
+    public Long getId() {
+        return id;
+    }
+
+    public void setId(Long id) {
+        this.id = id;
+    }
+
+    public Integer getCapacity() {
+        return capacity;
+    }
+
+    public void setCapacity(Integer capacity) {
+        this.capacity = capacity;
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public void setDescription(String description) {
+        this.description = description;
+    }
+
+    @Override
+    public String toString() {
+        final StringBuilder sb = new StringBuilder("CinemaHall{");
+        sb.append("id=").append(id);
+        sb.append(", capacity=").append(capacity);
+        sb.append(", description='").append(description).append('\'');
+        sb.append('}');
+        return sb.toString();
```

Comment:
it is not practiced to use stringBuilder in toString() - here we can do concatenation